spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -84,9 +84,9 @@ def resolve(path, http_verb=:get)
       posts_path(locale: :fr, :i18n_posts => 'puestos').should eql '/fr/messages'
     end
 
```

Comment:
It would probably be good to have an additional test here that other params are appended correctly

```
    it 'should append any other params' do
      posts_path(locale: :fr, :i18n_posts => 'puestos', :foo => 'bar').should eql '/fr/messages?foo=bar'
    end
```